src/Components/Server/test/Circuits/RevalidatingServerAuthenticationStateProvider.cs

```diff
@@ -184,7 +185,8 @@ public async Task SuppliesCancellationTokenThatSignalsWhenRevalidationLoopIsBein
         }
 
         [Fact]
-        public async Task IfValidateAuthenticationStateAsyncReturnsUnrelatedCancelledTask_TreatAsFailure()
+        [QuarantinedTest]
+        public async Task IfValidateAuthenticationStateAsyncReturnsUnrelatedCanceledTask_TreatAsFailure()
```

Comment:
Ran into this flaky test while working on #10759 and have created an issue for it under #19940.